api/src/test/java/org/openmrs/validator/ConditionValidatorTest.java

```diff
@@ -51,16 +51,15 @@ public void setUp(){
 
 	@Test
 	public void shouldFailIfGivenNull(){
-		expectedException.expect(IllegalArgumentException.class);
-		expectedException.expectMessage(NULL_ERROR_MESSAGE);
-		validator.validate(null, errors);
+		IllegalArgumentException exception = assertThrows (IllegalArgumentException.class ,() -> validator.validate(null, errors));
```

Comment:
please check the whitespaces something seems to have gone wrong in your changes.

`assertThrows (IllegalArgumentException.class ,() -> validator.validate(null, errors));`
should be
`assertThrows(IllegalArgumentException.class, () -> validator.validate(null, errors));`